### ISSUE TITLE
No module named '_pyo' again

### ISSUE THREAD

**Issue by zappfinger** [belangeo/pyo#145]
On my Windows system I finally had pyo running. (Python3.7, wxPython installed) Now I wanted to try the funnysynth example and I get:
C:\Python37\python.exe C:/Users/toosb/PycharmProjects/pyoo/funnysynth.py
Traceback (most recent call last):
  File "C:/Users/toosb/PycharmProjects/pyoo/funnysynth.py", line 3, in <module>
    from pyolib._wxwidgets import VuMeter, ControlSlider
  File "C:\Python37\lib\site-packages\pyolib\_wxwidgets.py", line 25, in <module>
    from ._core import rescale
  File "C:\Python37\lib\site-packages\pyolib\_core.py", line 58, in <module>
    from _pyo import *
ModuleNotFoundError: No module named '_pyo'

**Comment by belangeo** [comment]
How did you install pyo? These paths ("C:\Python37\lib\site-packages\pyolib_wxwidgets.py") look wrong for 0.9.2 version.

And what is funnysynth.py? 
A line like this:
```
from pyolib._wxwidgets import VuMeter, ControlSlider
```
Used to work with pyo prior to 0.9.2 but the structure of the library has changed to accomodate pip install. Now, it must be:
```
from pyo.lib._wxwidgets import VuMeter, ControlSlider
```

**Comment by zappfinger** [comment]
I installed it like you suggested in the thread 'Pyo on Windows':
py -3.7 -m pip install -i https://test.pypi.org/simple/ pyo
py -3.7 -m pip install -U wxPython

BTW, FunnySynth comes from this post:
https://groups.google.com/forum/#!topic/pyo-discuss/B8GfJQE-Slc

**Comment by belangeo** [comment]
Oh, I see, FunnySynth.py (didn't remember!) was the first draft of what become Zyne:

https://github.com/belangeo/zyne

I just tried this on my Win10 computer:

1) Uninstall pyo under python 3.7 (just to be sure)
```
py 3.7 -m pip uninstall pyo
```
2) Install from test.pypi:
```
py -3.7 -m pip install -i https://test.pypi.org/simple/ pyo
```
3) Clone and install pyo-tools (needed by Zyne)
```
git clone https://github.com/belangeo/pyo-tools.git
cd pyo-tools
py -3.7 setup.py install
cd ..
```
4) Clone and run Zyne:
```
git clone https://github.com/belangeo/zyne.git
cd zyne
py -3.7 Zyne.py
```

Everything runs fine. The graphical interface of Zyne needs some fine-tuning on windows because of the switch from wxpython classic to wxpython Phoenix but I can play with it and get sound. Can you try this on your computer? Maybe you'll have to try different Output Driver and Midi interface to get it working properly...

**Comment by zappfinger** [comment]
That was not so succesful:
C:\Users\toosb>py -3.7 -m pip uninstall pyo
Cannot uninstall 'pyo'. It is a distutils installed project and thus we
cannot accurately determine which files belong to it which would lead to
only a partial uninstall.

C:\Users\toosb>py -3.7 -m pip install -I -i https://test.pypi.org/simple/
pyo
Looking in indexes: https://test.pypi.org/simple/
Collecting pyo
  Using cached
https://test-files.pythonhosted.org/packages/ab/41/5e431bee23745cee9003c5f70f113b6b5c21bd8715b0c73ddabfffceea65/pyo-0.9.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File
"C:\Users\toosb\AppData\Local\Temp\pip-install-r9rry5ch\pyo\setup.py", line
204, in <module>
        [os.path.join(data_files_common_path, f) for f in
os.listdir(data_files_common_path) if f.endswith(".dll")]),
    FileNotFoundError: [WinError 3] Het systeem kan het opgegeven pad niet
vinden: 'win32dlls\\win32_pyo_data_files_common'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in
C:\Users\toosb\AppData\Local\Temp\pip-install-r9rry5ch\pyo\

Op zo 14 apr. 2019 om 02:30 schreef Olivier Bélanger <
notifications@github.com>:

**Comment by zappfinger** [comment]
I did get Zyne running on OS X now, after I was able to finally install wxPython for my default Python.
It runs fine, despite there is a small syntax error:
Traceback (most recent call last):
  File "/Users/richard/PycharmProjects/pyoTest/zyne/Zyne.py", line 754, in MacOpenFile
    self.frame.openfile(filename)
  File "/Users/richard/PycharmProjects/pyoTest/zyne/Zyne.py", line 663, in openfile
    dic = eval(text)
  File "<string>", line 2
    import wx, os, sys
         ^
SyntaxError: invalid syntax

**Comment by belangeo** [comment]
> C:\Users\toosb>py -3.7 -m pip install -I -i https://test.pypi.org/simple/ pyo

Oh, I did this before I uploaded a new version for MacOS (0.9.3), so pip grabbed the windows wheel for 0.9.2. From your log, pip tried to compile 0.9.3 (which is a pain on windows!) because I don't uploaded windows wheels yet. You can try to force the version number:

`py -3.7 -m pip install -I -i https://test.pypi.org/simple/ pyo==0.9.2`

**Comment by belangeo** [comment]
> File "/Users/richard/PycharmProjects/pyoTest/zyne/Zyne.py", line 663, in openfile
> dic = eval(text)
> File "", line 2
> import wx, os, sys
> ^
> SyntaxError: invalid syntax

Ok, I can reproduce... Will fix it! Thanks!

**Comment by belangeo** [comment]
0.9.3 is now available for windows (32-bit and 64-bit) on test.pypi.org.

**Comment by danieljbradley** [comment]
I have a similar issue but no underscore before "pyo":
`
ModuleNotFoundError: No module named 'pyo'`

I installed pyo_0.9.1_py3.6_setup.exe : Windows (Win 7 and higher), python 3.6, 32-bit

Does it matter than I have 64 bit version of python installed (via Anaconda)?  (I uninstalled from C:\ and reinstalled in Anaconda directory but that did not help.)  I checked the environment variables and it looked like pyo was in the path variable.

Thanks in advance for your help!
